apis/python/test/test_index.py

```diff
@@ -350,20 +350,6 @@ def test_index_with_incorrect_num_of_query_columns_complex(tmp_path):
                     with pytest.raises(TypeError):
                         index.query(query, k=1)
 
-                # TODO(paris): This will throw with the following error. Fix and re-enable, then remove
```

Comment:
This was added by Andrew accidentally in a merge conflict: https://github.com/TileDB-Inc/TileDB-Vector-Search/commit/ebcf35b94377bd07b93cfd0a715043938e636364#diff-d6acf5a535e6bc7ca26f487ae25cecf98fe5244e93d1f44367a7a6bbef5ef91eR183